src/legacy/core_plugins/interpreter/public/functions/tsvb.js

```diff
@@ -64,10 +64,17 @@ export const tsvb = () => ({
       uiState: uiState,
     });
 
+    response.visType = 'metrics';
```

Comment:
is this needed downstream? i don't see anywhere that `visData.visType` is referenced, but may be missing something.

Reply:
yeah, check visualization renderer